.github/workflows/ci.yml

```diff
@@ -26,37 +26,64 @@ jobs:
     name: Node.js v${{ matrix.NodeVersionDisplayName }} (${{ matrix.OS }})
     runs-on: ${{ matrix.OS }}
     steps:
+      - name: Create ~/.rush-user/settings.json
+        shell: pwsh
+        run: |
+          mkdir -p $HOME/.rush-user
+          @{ buildCacheFolder = Join-Path ${{ github.workspace }} rush-cache } | ConvertTo-Json > $HOME/.rush-user/settings.json
```

Comment:
Can you add a comment explaining what this is doing?  Not everybody is familiar with PowerShell particularly with the `${{` macro superimposed on its syntax.

Something like:

```ps
# Create a .rush-user/settings.json file containing a configuration such as this:
#
#   { "buildCacheFolder":  "/home/runner/work/rushstack/rushstack/rush-cache" }
#
# This is... ❓❓❓
```

Actually I have no idea what it is.  `.rush-user/settings.json` is not documented anywhere on the website.  Are users supposed to create it? Is this an internal implementation detail? What is this CI step doing?

Reply:
Did that never get documented? It's how users configured a machine-wide Rush cache location.